src/platforms/dotnet/profiling/index.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Set Up Profiling
+description: "Learn how to enable profiling in your app if it is not already set up."
+sidebar_order: 5000
+---
+
+<PlatformContent includePath="profiling/index/preface" />
+
+With [profiling](/product/profiling/), Sentry allows you to collect and analyze performance profiles from real user devices in production to give you a complete picture of how your application performs in a variety of environments.
+
+<Note>
+
+.NET profiling alpha is available .NET 6.0+ (tested on .NET 7.0 & .NET 8.0 as well) for:
+- Windows
+- Linux
+- macOS
+- iOS
+- macCatalyst
+
+</Note>
+
+## Enable Profiling
+
+To enable profiling, set the `ProfilesSampleRate`. 
+Additionally, for all platforms except iOS/Mac Catalyst, you need to add a dependency on the `Sentry.Profiling` NuGet package.
+
+Profiling depends on Sentry’s performance monitoring product being enabled beforehand. To enable performance monitoring in the SDK, set the `TracesSampleRate` option to the desired value.
+
+```csharp {tabTitle:iOS/macCatalyst}
+using Sentry;
+
+SentrySdk.Init(options =>
+{
+    // ... usual setup options omitted for clarity (see Getting Started) ...
+    
+    // Sample rate for your transactions, e.g. value 0.1 means we want to report 10% of transactions. 1.0 is 100%
+    options.TracesSampleRate = 1.0;
+    
+    // Sample rate for profiling, applied on top of othe TracesSampleRate, 
+    // e.g. 0.2 means we want to profile 20 % of the captured transactions, i.e. 2 % of all transactions.
+    options.ProfilesSampleRate = 0.2;
+});
+```
+
+```csharp {tabTitle:Windows/Linux/macOS}
+using Sentry;
+
+SentrySdk.Init(options =>
+{
+    // ... usual setup options omitted for clarity (see Getting Started) ...
+    
+    // Sample rate for your transactions, e.g. value 0.1 means we want to report 10% of transactions.
+    options.TracesSampleRate = 0.1;
+    
+    // Sample rate for profiling, applied on top of othe TracesSampleRate, 
+    // e.g. 0.2 means we want to profile 20 % of the captured transactions, i.e. 2 % of all transactions.
+    options.ProfilesSampleRate = 0.2;
+    
+    // Requires NuGet package: Sentry.Profiling
+    // Note: by default, the profiler is initialized asynchronously, this can be tuned by passing an desired initialization timeout to the constructor.
+    options.AddIntegration(new ProfilingIntegration());
```

Comment:
How about we show the code that blocks to show how it works?
```suggestion
    options.AddIntegration(new ProfilingIntegration(
        // During startup, wait up to 500ms to profile the app startup code. This could make launching the app a bit slower so comment it out if your prefer profiling to start asynchronously
        TimeSpan.FromMilliseconds(500)
    ));
```